internal/services/formatters/ruby/bundler/formatter.go

```diff
@@ -109,23 +112,40 @@ func (f *Formatter) removeOutputEsc(output string) string {
 	return output
 }
 
-func (f *Formatter) parseOutput(output, projectSubPath string) {
+func (f *Formatter) parseOutput(output, projectSubPath string) error {
+	done, err := f.isValidOutput(output)
+	if done {
```

Comment:
I think that `done` should be renamed to `isInvalid`

Also I think that the check should be `if isInvalid || err != nil`

Reply:
err can be different of nil only when isInvald == true, so there's no need to check that, i'll  rename it.

Reply:
Make sense, so I think that we can resolve this conversation.